.gitignore

```diff
@@ -5,7 +5,7 @@
 /tmp/
 
 # dependencies
-/node_modules/
+**/node_modules/
```

Comment:
I don't think we should end up with nested node_modules folders, what problem is this solving?